### PR TITLE
Release 7.0.0

### PR DESCRIPTION
**Pull request by crow**
Release 7.0.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,17 @@
 UA Unity ChangeLog
 ==================
 
+Version 7.0.0 - March 6, 2020
+=============================
+- Updated Android Airship SDK to 12.2.0.
+- Updated iOS Airship SDK to 13.1.0.
+- Added screen tracking to iOS and Android.
+- Improved Message Center on iOS and Android.
```
Comment (rlepinski) [inline CHANGELOG.md]: how did we improve this? copy and paste from something else?

Comment (crow) [reply CHANGELOG.md]: https://github.com/urbanairship/ua-unity-plugin/pull/97

**Comment by crow** [comment]
Waiting on MOBILE-1331 and MOBILE-1329

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,10 +1,18 @@
 UA Unity ChangeLog
 ==================
 
+Version 7.0.0 - March 6, 2020
+=============================
+- Updated Android Airship SDK to 12.2.0.
+- Updated iOS Airship SDK to 13.1.0.
+- Added screen tracking.
+- Improved Message Center.
```
Comment (rlepinski) [inline CHANGELOG.md]: We do not improve message center.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,10 +1,18 @@
...
+- Added support for channel attributes.
```
Comment (rlepinski) [inline CHANGELOG.md]: Also added methods to edit in-app automation display interval, a way to pause/resume IAA, and data privacy controls.

File: CHANGELOG.md (hunk 2 above)
Comment (crow) [reply CHANGELOG.md]: https://github.com/urbanairship/ua-unity-plugin/pull/97

**Pull request merged**